index.js

```diff
@@ -182,6 +184,7 @@ class GithubScm extends Scm {
                 });
 
                 repoFullName = repo.data.full_name;
+                defaultBranch = repo.data.default_branch;
```

Comment:
I was curious how we get `default branch`, and I guess this is the key that github already has a field `repo.data.default_branch` to do so. Very handy!

Reply:
correct